src/app/api/services/dot-current-user/dot-current-user.service.spec.ts

```diff
@@ -55,6 +60,42 @@ describe('DotCurrentUserService', () => {
         });
     });
 
+    it('should get user Permissions data', () => {
+        const response = {
+            STRUCTURES: { canRead: true, canWrite: true },
+            HTMLPAGES: { canRead: true, canWrite: true },
+            TEMPLATES: { canRead: true, canWrite: true },
+            CONTENTLETS: { canRead: true, canWrite: true }
+        };
+        const userId = 'test';
+        dotCurrentUserService
+            .getUserPermissions(userId)
+            .subscribe((permissions: DotPermissionsType) => {
+                expect(permissions).toEqual(response);
+            });
+
+        const req = httpMock.expectOne(
+            `v1/permissions/_bypermissiontype?userid=${userId}&permission=&permissiontype=`
```

Comment:
Is normal that the url ends up like `&permissiontype=`? without a value?

Reply:
yes the BE will get it as empty params, so no problems